docs/msbuild/customize-your-local-build.md

```diff
@@ -18,7 +18,12 @@ When you work in a team that used a code repository like GitHub, source control,
 
 ## .user file
 
-*Microsoft.Common.CurrentVersion.targets* imports `$(MSBuildProjectFullPath).user` if it exists, so you can create a file next to your project with that additional extension. For long-term changes you plan to check into source control, prefer changing the project itself, so that future maintainers do not have to know about this extension mechanism.
+Using `$(MSBuildProjectFullPath).user`, also referred as `.user` file in this context, is also an option. This file is intended to keep extensions, options, or variables that are specific to your local machine. It is not intended to be uploaded to source control, and it is automatically checked on `.gitignore`. For more extensive changes prefer changing the project itself, so that future maintainers do not have to know about this extension mechanism.
+
+On supported multitargeted projects the `.user` file is automatically imported in inner builds and outer builds, so you can just create the file within the solution. If you are working on another type of build, you can still use the `.user` file. You can create it within your solution and then import it in your project file.
```

Comment:
"inner" and "outer" builds are not generally user-facing concepts, so I'm not sure this is the best way to phrase this. @ghogen, what happened is that prior to 17.9, the `.user` file was imported only for the inner build, so it wasn't usable to tweak things like packaging. @maridematte made it more consistent for 17.9+. See https://github.com/dotnet/msbuild/issues/9131 / https://github.com/dotnet/msbuild/pull/9558. Can you think of a way to say that that's a bit more friendly?